odoo/tools/image.py

```diff
@@ -1,32 +1,22 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
 import base64
 import binascii
 import io
-from typing import Tuple, Union
-
-from PIL import Image, ImageOps
-# We can preload Ico too because it is considered safe
-from PIL import IcoImagePlugin
-try:
-    from PIL.Image import Transpose, Palette, Resampling
-except ImportError:
-    Transpose = Palette = Resampling = Image
-
 from random import randrange
 
+from odoo._monkeypatches.PIL import patch_module as patch_PIL
```

Comment:
To resolve the circular import, you could test here that `odoo._monkeypatches` is in `sys.modules` - and leave a comment.

Reply:
:vomiting_face: